src/SignalR/common/Http.Connections/src/ConnectionEndpointRouteBuilderExtensions.cs

```diff
@@ -125,6 +126,9 @@ public static ConnectionEndpointRouteBuilder MapConnections(this IEndpointRouteB
             {
                 e.Metadata.Add(data);
             }
+
+            // Add INonBrowserEndpointMetadata to indicate this is a non-browser endpoint (SignalR)
```

Comment:
```suggestion
            // Add IApiEndpointMetadata to indicate this is a non-browser endpoint (SignalR)
```

Reply:
Wonder if there are any blazor endpoints that should also be marked?

Reply:
https://github.com/dotnet/aspnetcore/blob/bf1f642be9fe230fed38e10561d4aa9ed5859b8f/src/Components/Server/src/Builder/ComponentEndpointRouteBuilderExtensions.cs#L78-L86

These are what I was thinking of @dotnet/aspnet-blazor-eng thoughts?